app/service/model/workspace_role.py

```diff
@@ -16,21 +16,32 @@ class WorkspaceRoleLib:
     from app.utils.session import session_hook
 
     def __init__(self):
-        self.roles = ["admin", "reviewer"]
+        self.roles: [] = ["admin", "reviewer"]
 
     def run(self):
-        for role in self.roles:
-            WorkspaceRoleLib.create(data={"name": role})
+        from app.logs import logger
+
+        WorkspaceRoleLib.create(names=self.roles)
+        logger.info(f"Done Creating customer roles")
 
     @staticmethod
     @session_hook
-    def create(db: Session, data: dict):
-        workspace_role = WorkspaceRole(**data)
+    def create(db: Session, names: []):
```

Comment:
This should be rename to bulk_create and change name to records
Then use bulk_insert